packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -201,6 +205,10 @@ export class CeramicCliUtils {
       }
       if (network) {
         config.network.name = network
+        const getIndexingConfig = getNetworkDefaultConfig(network)
+        config.indexing.allowQueriesBeforeHistoricalSync =
```

Comment:
```suggestion
        config.indexing.allowQueriesBeforeHistoricalSync = config.indexing.allowQueriesBeforeHistoricalSync ?? getIndexingConfig.allowQueriesBeforeHistoricalSync
```

Reply:
although I'd also like to rename `getIndexingConfig` to something like `defaultIndexingConfig`